UefiTestingPkg/AuditTests/PagingAudit/UEFI/X64/PagingAuditProcessor.c

```diff
@@ -769,12 +771,16 @@ DumpPlatforminfo (
     PhaseString = "DXE";
   }
 
+  MemoryAttributeProtocol = NULL;
+  gBS->LocateProtocol (&gEfiMemoryAttributeProtocolGuid, NULL, (VOID **)&MemoryAttributeProtocol);

```

Comment:
Same comment as in PagingAuditProcessor.c. I think this should check that EFI_NOT_FOUND is not returned in addition to checking whether the point is non-null to consider finding the attribute successful.

Reply:
Updated to set MemoryAttributeProtocol to NULL if the locate protocol call returns an error.